docs/interacting-with-fleet-manager.md

```diff
@@ -0,0 +1,102 @@
+Red Hat SSO (sso.redhat.com) by default. Assuming the default OIDC authentication server is
+being used, this can be performed by interacting with the OCM API. This can be
+easily done interacting with it through the [ocm cli](https://github.com/openshift-online/ocm-cli/releases)
+and retrieve OCM tokens using it. To do so:
+1. Login to your desired OCM environment via web console using your Red Hat
+   account credentials. For example, for the OCM production environment, go to
+   https://console.redhat.com and login.
+1. Get your OCM offline token by going to https://console.redhat.com/openshift/token
+1. Login to your desired OCM environment through the OCM CLI by providing the
+   OCM offline token and environment information:
+   ```
+   ocm login --token <ocm-offline-token> --url <ocm-api-url>
+   ```
+   `<ocm-api-url>` is the URL of the OCM API. Some shorthands can also
+   be provided like `production` or `staging`
+1. Generate an OCM token by running: `ocm token`. The generated token is the token
+   that should be used to perform a request to the Fleet Manager API. For example:
+  ```
+  curl -H "Authorization: Bearer <result-of-ocm-token-command>" http://127.0.0.1:/8000/api/kafkas_mgmt
+  ```
+  OCM tokens other than the OCM offline token have an expiration time so a
+  new one will need to be generated when that happens
+
+There are some additional steps needed if you want to be able to perform
+certain actions that have additional requirements. See the
+[_User Account & Organization Setup_](getting-credentials-and-accounts.md#user-account--organization-setup) for more information
+
+### Creating a Kafka Request
+
+The following example shows how to create a Kafka Request in AWS:
+```
+curl -v -XPOST -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas?async=true -d '{ "region": "us-east-1", "cloud_provider": "aws",  "name": "test-kafka", "multi_az":true}'
+```
+
+#### Listing a Kafka Request
+
+The following example shows how to List a Kafka Request:
+```
+curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_request_id> | jq
+```
+
+### Listing all Kafka Requests
+The following example shows how to List all Kafka Requests:
+```
+curl -v -XGET -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas | jq
+```
+
+### Deleting a Kafka Request
+The following example shows how to delete a Kafka Request
+```
+curl -v -X DELETE -H "Authorization: Bearer $(ocm token)" http://localhost:8000/api/kafkas_mgmt/v1/kafkas/<kafka_request_id>?async=true
+```
+
+### Using the Kafka Admin Server API
+
+The Kafka Admin Server API is used for managing topics, acls, and consumer groups
+on a given Kafka Request.
+
+The Kafka Admin Server OpenAPI specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/src/main/resources/openapi-specs/kafka-admin-rest.yaml).
```

Comment:
```suggestion
The Kafka Admin Server OpenAPI specification can be found [here](https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/.openapi/kafka-admin-rest.yaml).
```

the link doesn't exist. I am suggesting a different one but would need @MikeEdgar to confirm if it is the current used openapi file for the admin server.

Reply:
thanks @MikeEdgar

Reply:
updated :+1: